aptos-move/framework/aptos-framework/sources/account.move

```diff
@@ -357,6 +359,7 @@ module aptos_framework::account {
     ) acquires Account, OriginatingAddress {
         let addr = signer::address_of(account);
         assert!(exists_at(addr), error::not_found(EACCOUNT_DOES_NOT_EXIST));
+        permissioned_signer::assert_master_signer(account);
```

Comment:
Could you explain here or add a doc in the PR description on when this assert is needed in the `account` module.